src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import * as Sentry from '@sentry/browser';
```

Comment:
Can't we just do

```
import { captureException } from '@sentry/browser';?
```
since this is the only method used there?

Reply:
I think it needs to load other stuff in global scope to catch uncaught exceptions.